authors/apps/authentication/views.py

```diff
@@ -49,7 +49,7 @@ def post(self, request):
 
 
 class UserRetrieveUpdateAPIView(RetrieveUpdateAPIView):
-    permission_classes = (IsAuthenticated,)
+    permission_classes = (IsAuthenticated, )
```

Comment:
@lytes20 
- Was this change necessary?

Reply:
@ssewilliam, That was probably caused by him performing PEP8 formating.

Reply:
no, it wasn't, correcting this now